Detectors/Upgrades/IT3/base/include/ITS3Base/SegmentationSuperAlpide.h

```diff
@@ -0,0 +1,187 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file SegmentationSuperAlpide.h
+/// \brief Definition of the SegmentationSuperAlpide class
+
+#ifndef ALICEO2_ITS3_SEGMENTATIONSUPERALPIDE_H_
+#define ALICEO2_ITS3_SEGMENTATIONSUPERALPIDE_H_
+
+#include <Rtypes.h>
+#include "MathUtils/Cartesian.h"
+#include "CommonConstants/MathConstants.h"
+#include "ITSMFTBase/SegmentationAlpide.h"
+
+namespace o2
+{
+namespace its3
+{
+
+/// Segmentation and response for pixels in ITSMFT upgrade
+/// Questions to solve: are guardrings needed and do they belong to the sensor or to the chip in
+/// TGeo. At the moment assume that the local coord syst. is located at bottom left corner
+/// of the ACTIVE matrix. If the guardring to be accounted in the local coords, in
+/// the Z and X conversions one needs to first subtract the  mGuardLeft and mGuardBottom
+/// from the local Z,X coordinates
```

Comment:
in the SegmentationAlpide the guardring is the part of the sensor (epitaxial layer) and is accounted in coordinates transformation, as you also did

Reply:
We are not sure if we understand you correctly, but we just copied this comment from SegmentationAlpide. So if it is wrong here, it isalso wrong there.

Reply:
OK, I did not realize that I am commenting on my comment, then the logic/math for pixel<->local coord formally is correct. But I see that you are defining the PassiveEdgeReadOut in a same way as for the Alpide, i.e. it is a dead zone along Z (columns count). Isn't cylindrical layer readout done from the side(s)? If so, then the meaning of top/bottom and side guardrings should be swapped as well the meaning of row and column.

Reply:
Thanks for noticing! We have set the passive edge to zero for the time being. We still do not know how the passive areas will look like in the final detector and we leave this detail to future implementations.